src/app/globals.css

```diff
@@ -1,26 +1,60 @@
 @import "tailwindcss";
-
-:root {
-  --background: #ffffff;
-  --foreground: #171717;
-}
+@import "tw-animate-css";
 
 @theme inline {
-  --color-background: var(--background);
-  --color-foreground: var(--foreground);
-  --font-sans: var(--font-geist-sans);
-  --font-mono: var(--font-geist-mono);
-}
+  --color-black: #222222;
+  --color-blue: #41415f;
+  --color-red: #a7271d;
+  --color-orange: #c57564;
+  --color-yellow-white: #efebd2;
+  --color-yellow: #f4d590;
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background: #0a0a0a;
-    --foreground: #ededed;
-  }
+  --type-display-large: 3.5625rem / 4rem var(--font-bai);
+  --type-display-medium: 2.8125rem / 3.25rem var(--font-bai);
+  --type-display-small: 2.25rem / 2.75rem var(--font-bai);
+  --type-headline-large: 2rem / 2.5rem var(--font-bai);
+  --type-headline-medium: 1.75rem / 2.25rem var(--font-bai);
+  --type-headline-small: 1.5rem / 2rem var(--font-bai);
+  --type-title-large: 1.375rem / 1.75rem var(--font-bai);
+  --type-title-medium: 1rem / 1.5rem var(--font-bai);
+  --type-title-small: 0.875rem / 1.25rem var(--font-bai);
+  --type-label-large: 0.875rem / 1.25rem var(--font-bai);
+  --type-label-medium: 0.75rem / 1rem var(--font-bai);
+  --type-label-small: 0.6875rem / 1rem var(--font-bai);
+  --type-body-large: 1rem / 1.5rem var(--font-bai);
+  --type-body-medium: 0.875rem / 1.25rem var(--font-bai);
+  --type-body-small: 0.75rem / 1rem var(--font-bai);
+  --type-chinese-large: 7rem / 4rem var(--font-kart);
+  --type-chinese-medium: 4.5rem / 3rem var(--font-kart);
+  --type-chinese-small: 2.25rem / 2.25rem var(--font-kart);
+  --type-chinese-cursive: 0.6rem / 6rem var(--font-kart);
+
+  --tracking-display-large: -0.25px;
+  --tracking-display-medium: 0;
+  --tracking-display-small: 0;
+  --tracking-headline-large: 0;
+  --tracking-headline-medium: 0;
+  --tracking-headline-small: 0;
+  --tracking-title-large: 0;
+  --tracking-title-medium: 0.15px;
+  --tracking-title-small: 0.1px;
+  --tracking-label-large: 0.1;
+  --tracking-label-medium: 0.5px;
+  --tracking-label-small: 0.5px;
+  --tracking-body-large: 0.5;
+  --tracking-body-medium: 0.25px;
+  --tracking-body-small: 0.4px;
+  --tracking-chinese-large: -3.5px;
+  --tracking-chinese-medium: -2px;
+  --tracking-chinese-small: -1px;
+  --tracking-chinese-cursive: -20px;
```

Comment:
You can only omit `px` for `0`, everything else needs a unit! Please have another look.